grading/ssh-sudoer/bin/_run_student_intern

```diff
@@ -0,0 +1,95 @@
+#! /bin/python3.6
```

Comment:
Same remark that for ``bin/INGInious`` here: we won't maintain two copies with slight changes. If the behaviour differs, it would be better to define hooks and call them from the main ``_run_student_intern`` script.